docs/governance/governance.md

```diff
@@ -48,7 +48,35 @@ The “call for objections” or the vote must be conducted during a live TSC me
 
 ### Maintainers: <a href="#maintainers" id="maintainers"></a>
 
-Contributors who have the ability to commit code and contributions to a repo's main branch on the Baseline Protocol. A Contributor may become a Maintainer by a majority approval of the existing Maintainers. The initial number of maintainers required to merge a pull request to master in the github repo shall be three, but may be amended to no fewer than two by a simple majority of the maintainers.
+Baseline Maintainers have advanced code repository permissions and responsibilities. 
+
+#### Becoming a Baseline Maintainer
+
+First, one must attend at least two consecutive Baseline Core Dev meetings and contribute to the project through PRs, workgroups, or other contributions.
+
+Then complete the [Pull_Request_Template](https://github.com/eea-oasis/baseline/blob/main/.github/PULL_REQUEST_TEMPLATE.md) in its entirety to add your Github ID to the [Code Owners file](https://github.com/eea-oasis/baseline/blob/main/.github/CODEOWNERS).The current Maintainers will be assigned to review the Pull Request, and the PR will be merged through the PR approval process.
```

Comment:
Will any request to become a maintainer be accepted, if they meet the criteria? 
So there is no longer a majority approval of the existing Maintainers required?

Reply:
Yes and yes. I will explain in the PGB call this week.

> It would replace 'majority approval' with the existing process which is 'merged PR to add GH name to Code Owners File'. 
> 
> Before there was not a formal process in place for becoming a maintainer which would have had to include: making a request (how?) > maintainers conducting vote (how?) > vote being documented > repo permissions being updated.
> 
> And anyone was able to submit a PR to add their name to the Code Owners File (which requires 2 maintainer reviews for merging) which resulted in conflicting rules and a way to bypass the majority vote.
> 
> For the last months, the Baseline Protocol Core Devs & Maintainers have been operating using the Code Owners File because this removes all manual steps for voting or updating permissions, and keeps a public track record of the PRs. A Baseline Maintainer & TSC Member updated [the PR template](https://github.com/eea-oasis/baseline/blob/main/.github/PULL_REQUEST_TEMPLATE.md) for this type of change. 
>